### ISSUE TITLE
tflite's  output is far from tf's

### ISSUE THREAD

**Issue by MaeThird** [tensorflow/tensorflow#20117]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: N
- **OS Platform and Distribution**: Linux 4.16.13-2-ARCH
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**:'v1.8.0-3238-g52bf2fe0f6' 1.9.0-rc0
- **Python version**:  3.6.5
- **Bazel version (if compiling from source)**: v0.12.0
- **GCC/Compiler version (if compiling from source)**: gcc-7.1
- **CUDA/cuDNN version**: cuda-9.2,cuDNN-7.1
- **GPU model and memory**: 16G
- **Exact command to reproduce**:
###
I trained my model on mobilenet_v2-ssd with codes in models/research/object_detection/ ,export and convert the model from node `Preprocessor/sub` to `concat,concat_1` for the reason that tflite doesn't support some ops.
**With the same input image and same preprocessing** I print out the values of `concat_` whose shape is [1,1917,2] (there is only one ground truth class in my dataset,so the last dimension is 1+1 == 2)
the tf's output is `[[[8.46199608 -9.81417]...`,but the tflite's is `[[[17.357746 -17.360010]...`
The only different obviously I can find is **the training network used  `FusedBatchNorm` 
but in the exported tflite model the `FusedBatchNorm` was replaced by `{'fused_activation_function': 'NONE'} | ADD (opcode=0)`**
Does this differ care? Anyone help?

**Comment by karmel** [comment]
@aselle -- mind taking a look?

**Comment by tensorflowbutler** [comment]
Nagging Assignees @aselle, @achowdhery: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by achowdhery** [comment]
The instructions for support are coming soon. Thanks for your patience

**Comment by achowdhery** [comment]
Here is the blog post for end-to-end object detection training and deployment with Tensorflow Lite:
https://medium.com/tensorflow/training-and-serving-a-realtime-mobile-object-detector-in-30-minutes-with-cloud-tpus-b78971cf1193

**Comment by MaeThird** [comment]
I have noticed that you just updated  an `export_tflite_ssd_graph.py` and I'm trying on it.Thanks for your work.

**Comment by apivovarov** [comment]
I found that quantized TFLite model is different from non-quantized TFLite model.

**Comment by apivovarov** [comment]
ok, got it. instead of RELU6 quantized model uses  Requantize_32to8 - where I can explicitly set new min/max
